app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class PostsController < ApplicationController
+  before_action :authenticate_user!, only: %i[create index]
+
+  def index
+    # @posts = Post.all
+    @posts = Post.all
```

Comment:
Posts need to be ordered by datetime, preferably most recently created post first.

[`REQUIRED`]